requirements.txt

```diff
@@ -6,4 +6,4 @@ toolz
 ruamel.yaml
 importlib_metadata
 pymysql
-sqlalchemy==1.3.*
```

Comment:
sqlalchemy 2 can be quite different from v.1, there is even a mihgration guide: https://docs.sqlalchemy.org/en/20/changelog/migration_20.html

Just unfreezing like this seems risky..

Reply:
Yes, I realised a little while ago. 😊 It was useful to test the dependencies at least: nice to see that chanjo-report / scout did build and run ok with this configuration. But chanjo would likely bug out without changes..